way_to_home/tests/unittests/place/testmodel.py

```diff
@@ -0,0 +1,111 @@
+"""
+Place model tests
+===============
+"""
+from decimal import Decimal
+
+from custom_user.models import CustomUser
+from django.test import TestCase, Client
+
+from place.models import Place
+
+
+class PlaceViewTest(TestCase):
+    """TestCase for providing Place model testing"""
+    def setUp(self):
+        """Method that provides preparation before testing Place model features."""
+        custom_user = CustomUser.objects.create(id=2, email='mymail@icloud.com', is_active=True)
+        custom_user.set_password('qwerty12345')
+        custom_user.save()
+
+        self.client = Client()
+        login = self.client.login(email='mymail@icloud.com', password='qwerty12345')
+        self.assertTrue(login)
+
+        self.place = Place.objects.create(
+            id=11,
+            longitude=49.842601,
+            latitude=23.968448,
+            address='Широка 34, 79052',
+            name='Дім',
+            stop_id=None,
+            user=custom_user
+        )
+
+    def test_get_by_id(self):
+        """Provide tests for `get_by_id` method of certain Place instance."""
+        expected_route = Place.objects.get(id=self.place.id)
+        actual_route = Place.get_by_id(obj_id=self.place.id)
```

Comment:
I think it must be `place` instead of `route` :)

Reply:
Yeap :)